src/core/Akka.Persistence.Tests/Serialization/MessageSerializerRemotingSpec.cs

```diff
@@ -23,12 +23,12 @@ internal class Configs
 akka.actor {
   serializers {
     my-payload = ""Akka.Persistence.Tests.Serialization.MyPayloadSerializer, Akka.Persistence.Tests""
-    my-payload2 = ""Akka.Persistence.Tests.Serialization.MyPayload2Serializer, Akka.Persistence.Tests""
     old-payload = ""Akka.Persistence.Tests.Serialization.OldPayloadSerializer, Akka.Persistence.Tests""
+    testserializer = ""Akka.Serialization.HyperionSerializer, Akka.Serialization.Hyperion""
```

Comment:
In fact. After talking this over with @alexvaluyskiy we are not sure why we have this Spec at all. It was disabled before. But it does not test any real-world scenario as far as we know. 
What are the real-world scenario's for sending the `Persistent` or `AtomicWrite` types across the wire.